backend/tests/ops_site/cans/test_controller.py

```diff
@@ -16,3 +17,11 @@ def test_CAN_fiscal_year_serializer_has_depth_of_one():
 
 def test_CAN_fiscal_year_serializer_returns_all_models_fields():
     assert CANFiscalYearSerializer.Meta.fields == "__all__"
+
+
+def test_Portfolio_serializer_has_depth_of_one():
+    assert PortfolioSerializer.Meta.depth == 1
+
+
+def test_Portfolio_serializer_returns_all_models_fields():
+    assert PortfolioSerializer.Meta.fields == "__all__"
```

Comment:
These went into the wrong file.  We want them in `backend/tests/ops_site/portfolios/test_controller.py`.

Reply:
ah, missed that